tools/pythonpkg/src/pandas/analyzer.cpp

```diff
@@ -345,8 +345,9 @@ LogicalType PandasAnalyzer::InnerAnalyze(py::handle column, bool &can_convert, b
 	// Keys are not guaranteed to start at 0 for Series, use the internal __array__ instead
 	auto pandas_module = py::module::import("pandas");
 	auto pandas_series = pandas_module.attr("core").attr("series").attr("Series");
+
 	if (py::isinstance(column, pandas_series)) {
-		column = column.attr("__array__")();
```

Comment:
Another change that was made in an attempt to unify the arrow/numpy codepaths here, should probably be removed just to avoid potential unforeseen problems caused by this change